packages/qwik/public.d.ts

```diff
@@ -72,3 +72,5 @@ export {
   version,
   withLocale,
 } from './dist/core-internal';
+
+export type * from './dist/core-internal';
```

Comment:
The reason I split the internal and public was so that we wouldn't have many internal types as public API.

So every export that we want to expose must be listed separately.